routes/module.js

```diff
@@ -5,5 +5,7 @@ const router = express.Router();
 
 router.get("/list", moduleController.showModule);
 router.post("/add", moduleController.addModule);
+router.post("/update:Id",moduleController.updateModule);
+router.delete("/delete:Id",moduleController.deleteModule);
```

Comment:
WHY IS THERE NO SLASHHHHHHH PUT SLASHHHHHHHHHHH
this should be /:id
also please make sure you make testcases and apidocs for this after fixing these changes